stubs/fpdf2/fpdf/graphics_state.pyi

```diff
@@ -20,17 +20,17 @@ class GraphicsStateMixin:
     @text_color.setter
     def text_color(self, v: DeviceGray | DeviceRGB) -> None: ...
     @property
-    def underline(self): ...
+    def underline(self) -> bool: ...
     @underline.setter
-    def underline(self, v) -> None: ...
+    def underline(self, v: bool) -> None: ...
     @property
-    def font_style(self): ...
+    def font_style(self) -> str | None: ...
```

Comment:
I don't think this can ever be `None` -- why do you think it might be? :)

```suggestion
    def font_style(self) -> str: ...
```

Reply:
Font_style  : Even i was not sure but i saw font_style: optional[str] =None  , and optional is either str or None and the default value of font_style is "". Therefore i thought ,None can be a return type or argument.

Reply:
True, but that's for a different class, and it looks to me like there are quite a few tests in the test suite that assume that `FPDF.font_style` will always be a `str`. So I think I'd go with `str` :)

Reply:
> Font_style

Are you taking below test suit?
https://github.com/PyFPDF/fpdf2/blob/master/test/fonts/test_set_font.py#L121-L144